api/schemas/indicatorCodeChangeDto.yaml

```
@@ -0,0 +1,16 @@
+title: Indicator code definition change DTO
+description: Describe code definition that should be created or change to existed code definition
+type: object
+additionalProperties: false
+properties:
+  code:
+    description: Indicator code
+    type: string
+    minLength: 1
+    maxLength: 1
```

```suggestion
  code:
    description: Indicator code
    type: string
    pattern: [0-9a-z]{1}
```
I think make sense to define pattern to validate code value. In this case we will check at once length and possible values

---

I can't find alphabetical codes in MARC specification. Do we need alphabetical here? Do we also want to allow `#`, or we just won't create an indicator in such case?

---

Yes you right there is no aphabetical. as for undefined it should be `/`. So pattern then will be `[0-9\/]{1}`